app/controllers/spree/komoju_controller.rb

```diff
@@ -37,12 +37,23 @@ def payment_captured!
       payment.complete! unless payment.completed?
     end
 
+    def refund_params
+      params[:data][:refunds].last
+    end
+
+    def refund_description
+      refund_params[:description].blank? ? "Komoju refund" : refund_params[:description]
+    end
+
+    def refund_amount
+      # Converts cents into dollars
+      ::Money.new(refund_params[:amount], "USD").to_f
```

Comment:
@project-eutopia I'm confused here, why is the amount in USD? The plugin is JPY only.